stories/feed.js

```diff
@@ -3,11 +3,24 @@ import Feed from "../packages/aurora-feed";
 import { Background } from "../packages/aurora-ui";
 import { Themed } from "../packages/aurora-theme";
 
+/**
```

Comment:
This is duplicated from another file. Could we avoid this?

Reply:
It's fine to duplicate in stories I think, they're just docs.

Reply:
Actually, maybe I'm wrong. I have an idea.

Reply:
Okay, I moved these into `aurora-persist`